lib/composer/phpunit/phpunit/src/Framework/Assert/Functions.php

```diff
@@ -1948,10 +1599,7 @@ function logicalAnd()
  */
 function logicalNot(Constraint $constraint)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$constraint". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)